Tools/autotest/sim_vehicle.py

```diff
@@ -766,23 +935,29 @@ def generate_frame_help():
             binary_basedir = "build/sitl-debug"
         else:
             binary_basedir = "build/sitl"
-        vehicle_binary = os.path.join(find_root_dir(), binary_basedir, frame_infos["waf_target"])
+        vehicle_binary = os.path.join(find_root_dir(),
+                                      binary_basedir,
+                                      frame_infos["waf_target"])
     else:
         vehicle_binary = os.path.join(vehicle_dir, cmd_opts.vehicle + ".elf")
 
     if not os.path.exists(vehicle_binary):
         print("Vehicle binary (%s) does not exist" % (vehicle_binary,))
         sys.exit(1)
 
-    start_vehicle(vehicle_binary, find_autotest_dir(), cmd_opts, frame_infos, location)
+    start_vehicle(vehicle_binary,
+                  find_autotest_dir(),
+                  cmd_opts,
+                  frame_infos,
+                  location)
 
 if cmd_opts.delay_start:
     progress("Sleeping for %f seconds" % (cmd_opts.delay_start,))
     time.sleep(float(cmd_opts.delay_start))
 
 try:
     if cmd_opts.no_mavproxy:
-        time.sleep(3)  # Just wait to output the last command after run_in_terminal_window.sh
+        time.sleep(3)  # output our message after run_in_terminal_window.sh's
```

Comment:
Again, the comment doesn't need to be in it's own line?

Reply:
If it wasn't annoying flake8 I didn't necessarily play with stuff.

In this case I think the comment belongs on the line - it's apologising for dodgily using `time.sleep(3)`